lib/dialects/abstract/query.js

```diff
@@ -484,11 +484,6 @@ AbstractQuery.prototype.checkLoggingOption = function() {
     console.log('DEPRECATION WARNING: The logging-option should be either a function or false. Default: console.log');
     this.options.logging = console.log;
   }
-
-  if (this.options.logging === console.log) {
-    // using just console.log will break in node < 0.6
-    this.options.logging = function(s) { console.log(s); };
-  }
```

Comment:
We really need to support such versions in 2016?:)
I removed it because this "fake function" accepts only one argument ```s``` and not allow to log timings (for all node versions) in default sequelize configuration.